tests/unit/s2n_cipher_suite_match_test.c

```diff
@@ -781,6 +781,161 @@ int main(int argc, char **argv)
             }
         }
 
+        /* Test equal preference grouping */
+        {
+            /**
+             * TODO: This cipher suite will be moved to s2n_cipher_preferences w/ it's own security policy.
+             *       For now, it's only used for testing purposes.
```

Comment:
I will just remove this comment. I didn't intend the comment to suggest that the test needs to use an official security policy. Rather, it's saying that the policy was defined in the test suite because it's just for testing.